File: TheLionsDen.Model/Responses/RoomImageResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TheLionsDen.Model.Responses
{
    public class RoomImageResponse
    {
        public int RoomImageId { get; set; }
        public byte[] Image { get; set; } 
        public int RoomTypeId { get; set; }
    }
}

File: the_lions_den_mobile/lib/model/room_image/room_image_response.dart
import 'package:json_annotation/json_annotation.dart';
part 'room_image_response.g.dart';

@JsonSerializable()
class RoomImageResponse {
  int? roomImageId;
  int? roomTypeId;
  String? image;

  RoomImageResponse();

  factory RoomImageResponse.fromJson(Map<String, dynamic> json) =>
      _$RoomImageResponseFromJson(json);

  /// Connect the generated [_$RoomImageResponseToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RoomImageResponseToJson(this);
}
